lib/Gocdb_Services/Role.php

```diff
@@ -683,37 +699,60 @@ public function revokeRole(\Role $role, \User $callingUser){
     }
     /**
      * Check that removing a site role from a user would not leave an orphaned API credential:
-     * attached to a site where the user no longer has a active role.
-     * @param \Role $role
-     * @throws \Exception If revoking the Role would leave an APIAuthentication
-     *                    credential owned by a user without a role at the site
+     * attached to a site where the user no longer has a active role at the site or owning NGI.
+     * @param \Role $role       Role to be revoked
+     * @return array            Array keyed by site name with values of credential count at named
+     *                          site blocking revocation
```

Comment:
What do you mean by 'values of credential count at named site blocking revocation'? Do you mean the number of credentails at the site that are blocking the roles revokaiton?

If so, maybe reword the comment? It's taken me a while to parse what is going on.

Reply:
Yes, it took me a while too! Reworded.